### ISSUE TITLE
make cmd-nits broken on master

### ISSUE THREAD

**Issue by richsalz** [openssl/openssl#15250]
Does the project include a 'make cmd-nits' as part of its CI?

This is from my build wrapper:
```
/usr/bin/perl ./util/find-doc-nits -c
doc/man1/openssl-cmp.pod: cmp does not implement -keep_alive
Makefile:3870: recipe for target 'cmd-nits' failed
make: *** [cmd-nits] Error 1
```

Ping @DDvO

**Comment by DDvO** [comment]
> Does the project include a `make cmd-nits` as part of its CI?

Indeed, for some reason this is not done.

> `doc/man1/openssl-cmp.pod: cmp does not implement -keep_alive`

This will be filled in by #15053.

**Comment by mattcaswell** [comment]
This was fixed by #15053
